micropython/modules_py/interstate75.md

```diff
@@ -27,6 +27,7 @@ DISPLAY_INTERSTATE75_192X64
 DISPLAY_INTERSTATE75_256X64
 ```
 
+You are able to set which version of the Interstate75 that you are using with the `interstate75w=` as default this value is set to `False` to use the Interstate75 button layout by setting it to `True` the module will use the Intersate75W button layout. If you are using an Interstate75W it is important to set this value to `interstate75w=True`, otherwise this could disable the wifi in your project.
```

Comment:
Suggested edit:
"You are able to set which version of the Interstate75 that you are using with the `interstate75w=`. By default this value is set to `False` to use the Interstate75 button layout. By setting it to `True` the module will use the Interstate75W button layout. If you are using an Interstate75W it is important to set this value to `interstate75w=True`, otherwise this could disable the WiFi in your project."